src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -300,6 +305,14 @@ public static string SaveDOMAsHtml5(XmlDocument dom, string targetPath)
 			return targetPath;
 		}
 
+		private static void RemoveUiElements(XmlDocument dom)
```

Comment:
I'm pretty sure we already have code to remove UI elements somewhere. It's good to do it when saving the document...I'm surprised that isn't where it was already happening...but we don't want two versions of what it means to clean the UI stuff out of the document. I'd suggest finding the original code and adding a call (or enhance it if it was missing something).